frontend/src/components/metadata/EditMetadataDefinitionModal.tsx

```diff
@@ -0,0 +1,671 @@
+													}
+													label="Datasets"
+												/>
+												<FormControlLabel
+													control={
+														<Checkbox
+															checked={formInput.required_for_items.files}
+															onChange={() =>
+																handleMetadataRequiredInputChange("files")
+															}
+														/>
+													}
+													label="Files"
+												/>
+											</Grid>
+										</Grid>
+									</Tooltip>
+								</FormGroup>
+
+								{contextMap.map((item, idx) => {
+									return (
+										<Grid container>
+											<Grid item>
+												<ClowderInput
+													margin="normal"
+													fullWidth
+													required
+													id="metadata-context"
+													label="Term"
+													placeholder="Please enter context term"
+													value={item["term"]}
+													sx={{
+														mt: 1,
+														mr: 1,
+														alignItems: "right",
+														width: "300px",
+													}}
+													onChange={(event) => {
+														updateContext(idx, "term", event.target.value);
+													}}
+												/>
+											</Grid>
+											<Grid item>
+												<Autocomplete
+													id="metadata-auto-complete"
+													freeSolo
+													autoHighlight
+													inputValue={item["iri"]}
+													options={contextUrlMap["frequently_used"].map(
+														(option) => option.url
+													)}
+													onInputChange={(event, value) => {
+														updateContext(idx, "iri", value);
+													}}
+													renderInput={(params) => (
+														<ClowderInput
+															{...params}
+															required
+															label="IRI"
+															fullWidth
+														/>
+													)}
+													sx={{
+														mt: 1,
+														ml: 2,
+														alignItems: "right",
+														minWidth: "20em",
+													}}
+												/>
+											</Grid>
+											<IconButton
+												color="primary"
+												size="small"
+												onClick={() => addNewContext(idx)}
+											>
+												<AddBoxIcon />
+											</IconButton>
+											{idx == 0 ? (
+												<></>
+											) : (
+												<IconButton
+													color="primary"
+													size="small"
+													onClick={() => removeContext(idx)}
+												>
+													<DeleteOutlineIcon />
+												</IconButton>
+											)}
+										</Grid>
+									);
+								})}
+
+								<Button
+									variant="contained"
+									onClick={() => validateFormData(activeStep)}
+								>
+									Next
+								</Button>
+							</form>
+						</StepContent>
+					</Step>
+
+					{formInput.fields.map((input, idx) => {
+						return (
+							<Step key={idx}>
+								{idx == 0 ? (
+									<StepButton
+										color="inherit"
+										onClick={() => {
+											setActiveStep(idx + 1);
+										}}
+									>
+										<StepLabel>
+											Edit metadata entry*
```

Comment:
There is no delete entry option right next to the edit metadata
<img width="794" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/54859c8e-de51-4c23-b062-33b74aba4f0c">

Reply:
Done